### PR TITLE
#227 - adopt js page

### LINKED ISSUE #227: Adopt a JavaScripter page redesign
Redesigned the /adopt-a-jser page, to be more in tune with the rest of the redesigned pages.
Attached:
- mockups for desktop and mobile
- photos used

**FYI:** The text for the ADOPT part should be visible.
The text for the PHOTO CONTEST part should be hidden for now, until we decide to launch contest.

Here is the text in the mockups (it's slightly updated from what we had earlier this year):

**Learn About the Adopt a JavaScripter Couch-Surfing Program**

**Adopt a JavaScripter**

If you want to be "adopted" by one of our Cluj JavaScripters during the **JSHeroes conference**, you can use our "Adopt a JavaScripter" couch-surfing program.
We will pair you up with a **Host** or a **Guest** that we think is ok for you, according to the info you filled in the application forms.
Apply if you want to be a Guest, or if you are from Cluj-Napoca apply to be a Host.

**JSHeroes PHOTO CONTEST**

You are participating in the “Adopt a JavaScripter” couch-surfing program? Then enter the JSHeroes photo contest for a chance to win **2 free tickets** to the JSHeroes conference. 
How does the contest work?
Take a photo with yourself and your Host or Guest.
Then post it on Twitter with the hashtag **#adoptajavascripter**.
The winner will be selected randomly using random.org.
Post your photos until April 9, 2019.
Pretty simple!

adopt-page-redesign
adopt-page-mobile

jsheroes-adopt-jser-photo

adopt-photo

### PR DESCRIPTION
**Pull request by jonattfin**
Fix #227 

https://jsheroesio-eqrmrtdjwh.now.sh/adopt-a-jser

Responsiveness can be seen here: https://sizzy.co/?url=https%3A%2F%2Fjsheroesio-eqrmrtdjwh.now.sh%2Fadopt-a-jser (EG: for some reason it's working in Chrome only..)

Notes: 

1. I've changed a bit the image in the header to contain the center of the header image (with JSheroes & others logos). If you think it's not good, I will make it as it was in the original design.
2. The text for the photo contest is hidden / as per story said, but can be seen easily by removing the class "hidden-until-launch" from the parent div. just to see how it looks.
3. We've started using Zeplin with this story, I've already sent you invitations. 
The zeplin link for this design: https://app.zeplin.io/project/56d9947424589606059aa52b/

### CODE REVIEW

File: pages/adopt-a-jser.js [hunk 1]
```diff
@@ -1,206 +1,306 @@
...
-import AdoptAJSerHero from '../app/components/adopt-a-jser-hero';
 
-// add removed code in 2019
+import { mediaQueries, styles } from '../app/constants';
+
+function pixelsToEm(pixels) {
+  return `${(pixels / 16).toFixed(2)}em`;
+}
+
+const TwoSideButton = () => (
```
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: yes, it will be like that.

File: pages/adopt-a-jser.js [hunk 2]
```diff
@@ -1,206 +1,306 @@
...
+              Pretty simple!
+            </div>
+          </span>
+        </div>
+        <div className="image-box">
+          <picture>
+            <img
+              src="./static/img/adopt_a_jser_photo-contest.png"
+              className="image-wrapper"
+              alt=""
```
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: sure

File: pages/adopt-a-jser.js [hunk 3]
```diff
@@ -1,206 +1,306 @@
...
+        </div>
+      </div>
+      <div className="separation-line">&nbsp;</div>
+      <div className="container__item container__photo-contest">
+        <div className="text-box">
+          <span className="text-wrapper">
+            <div className="text-wrapper__title">JSHeroes PHOTO CONTEST</div>
+            <div>&nbsp;</div>
+            {/* The text for the PHOTO CONTEST part should be hidden for now, until we decide to launch contest. */}
+            <div className="hidden-until-launch">
```
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: i like your suggestion

File: pages/adopt-a-jser.js [hunk 4]
```diff
@@ -1,206 +1,306 @@
...
+            Couch-Surfing Program
+          </div>
+        </div>
+      </div>
+      <div className="container__item container__guest-or-host">
+        <div className="extend-with-border">
+          <TwoSideButton />
+        </div>
+        <div className="text-box">
+          <span className="text-wrapper">
```
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: good eyes, of course ;-)

[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: should be fine, now

File: pages/adopt-a-jser.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: should be fine

File: pages/adopt-a-jser.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: done

File: pages/adopt-a-jser.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonattfin) [reply pages/adopt-a-jser.js]: done, & clicking on the texts, you'll get to some google docs.

**Comment by jonattfin** [comment]
@ecris87 Can you review it again, Thx.

**Comment by ecris87** [comment]
Awesome, re-approved :)

**Pull request merged**